From e69a0afc0752da502ffa1f6046698478e8e47d9f Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Thu, 29 May 2025 08:58:32 +0300
Subject: [PATCH] [JSW-705] jgs: remove manual index from sw status

This patch removes manual index from progress
bits enum, to avoid overhead while adding new
enum to this. It can be added once before code
freeze.

Change-Id: I0162890923af34068ec50a2615822904e4cebfc2
---

diff --git a/arch/jgs/arc/progress_bits.h b/arch/jgs/arc/progress_bits.h
index 3ca8539..83ceb27 100644
--- a/arch/jgs/arc/progress_bits.h
+++ b/arch/jgs/arc/progress_bits.h
@@ -7,155 +7,152 @@
 #define _PROGRESS_BITS_H_
 
 enum boot_rom_progress {
-	PROGRESS_INIT = 0,
-	PROGRESS_INIT_ARC1_ICCM_INIT_FAILED = 1,
-	PROGRESS_INIT_SPI_MANIFEST_0_EXCEEDS_MAX_SIZE = 2,
-	PROGRESS_INIT_SPI_MANIFEST_0_DMA_START = 3,
-	PROGRESS_INIT_SPI_MANIFEST_0_CRC_FAIL = 4,
-	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_DEST = 5,
-	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_SIZE = 6,
-	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_IMG_SIZE = 7,
-	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_FLASH_OFFSET = 8,
-	PROGRESS_INIT_SPI_MANIFEST_0_DMA_ERROR = 9,
-	PROGRESS_INIT_SPI_MANIFEST_0_DMA_FINISH = 10,
-	PROGRESS_INIT_SPI_MANIFEST_0_HDR_MAGIC_FAIL = 11,
-	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_OFFSET = 12,
-	PROGRESS_INIT_SPI_MANIFEST_0_VERIFICATION_FAIL = 13,
-	PROGRESS_INIT_SPI_MANIFEST_0_INFO_SET_FAIL = 14,
-	PROGRESS_INIT_SPI_MANIFEST_0_DATA_SET_FAIL = 15,
-	PROGRESS_INIT_SPI_MANIFEST_0_INFO_GET_FAIL = 16,
-	PROGRESS_INIT_SPI_MANIFEST_0_DATA_GET_FAIL = 17,
-	PROGRESS_INIT_SPI_MANIFEST_0_FAIL = 18,
-	PROGRESS_INIT_SPI_MANIFEST_1_EXCEEDS_MAX_SIZE = 19,
-	PROGRESS_INIT_SPI_MANIFEST_1_DMA_START = 20,
-	PROGRESS_INIT_SPI_MANIFEST_1_CRC_FAIL = 21,
-	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_DEST = 22,
-	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_SIZE = 23,
-	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_IMG_SIZE = 24,
-	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_FLASH_OFFSET = 25,
-	PROGRESS_INIT_SPI_MANIFEST_1_DMA_ERROR = 26,
-	PROGRESS_INIT_SPI_MANIFEST_1_DMA_FINISH = 27,
-	PROGRESS_INIT_SPI_MANIFEST_1_HDR_MAGIC_FAIL = 28,
-	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_OFFSET = 29,
-	PROGRESS_INIT_SPI_MANIFEST_1_VERIFICATION_FAIL = 30,
-	PROGRESS_INIT_SPI_MANIFEST_1_INFO_SET_FAIL = 31,
-	PROGRESS_INIT_SPI_MANIFEST_1_DATA_SET_FAIL = 32,
-	PROGRESS_INIT_SPI_MANIFEST_1_INFO_GET_FAIL = 33,
-	PROGRESS_INIT_SPI_MANIFEST_1_DATA_GET_FAIL = 34,
-	PROGRESS_INIT_SPI_MANIFEST_1_FAIL = 35,
-	PROGRESS_INIT_SPI_VALID_MANIFEST_INDEX_FAIL = 36,
-	PROGRESS_INIT_EFUSE_INIT_FAIL = 37,
-	PROGRESS_INIT_EFUSE_SPI = 38,
-	PROGRESS_INIT_EFUSE_SPI_IMG_DESC_FAIL = 39,
-	PROGRESS_INIT_EFUSE_SPI_EXCEEDS_MAX_SIZE = 40,
-	PROGRESS_INIT_EFUSE_SPI_DMA_COPY_FAIL = 41,
-	PROGRESS_INIT_EFUSE_SPI_MAGIC_FAIL = 42,
-	PROGRESS_INIT_EFUSE_SPI_DESC_COUNT_FAIL = 43,
-	PROGRESS_INIT_EFUSE_SPI_DECODE_FAIL = 44,
-	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 45,
-	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 46,
-	PROGRESS_REVOKE_LOC_FAKE_FUSE = 47,
-	PROGRESS_KEY_LOC = 48, /* TODO: Remove & rearrange */
-	PROGRESS_KEY_LOC_ROM = 49,
-	PROGRESS_KEY_LOC_1 = 50, /* TODO: Remove & rearrange */
-	PROGRESS_VER_LOC = 51, /* TODO: Remove & rearrange */
-	PROGRESS_VER_LOC_ASIC_EFUSE = 52,
-	PROGRESS_VER_LOC_FAKE_FUSE = 53,
-	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND = 54,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_START = 55,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_VERSION_FAIL = 56,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_REVOKED = 57,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_HASH_FAIL = 58,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_AUTH_FAIL = 59,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_SUCCESS = 60,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_START = 61,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_VERSION_FAIL = 62,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_REVOKED = 63,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_HASH_FAIL = 64,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_AUTH_FAIL = 65,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_SUCCESS = 66,
-	PROGRESS_BRINGUP_PCI_PRI = 67,
-	PROGRESS_BRINGUP_PCI_PRI_NO_TMG_DESC = 68,
-	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID = 69,
-	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID = 70,
-	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG = 71,
-	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL = 72,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR = 73,
-	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE = 74,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE = 75,
-	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR = 76,
-	PROGRESS_BRINGUP_PCI_PRI_DMA = 77,
-	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL = 78,
-	PROGRESS_BRINGUP_PCI_PRI_VERIFY_SUCCESS = 79,
-	PROGRESS_BRINGUP_PCI_SEC = 80,
-	PROGRESS_BRINGUP_PCI_SEC_NO_IMG_DESC = 81,
-	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID = 82,
-	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID = 83,
-	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG = 84,
-	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL = 85,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR = 86,
-	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE = 87,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE = 88,
-	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR = 89,
-	PROGRESS_BRINGUP_PCI_SEC_DMA = 90,
-	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL = 91,
-	PROGRESS_BRINGUP_PCI_SEC_VERIFY_SUCCESS = 92,
-	PROGRESS_BOOT_ROM_TPM = 93,
-	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 94,
-	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 95,
-	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL = 96,
-	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL = 97,
- 	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC = 98,
-	PROGRESS_BOOT_ROM_TPM_FINISH = 99,
-	PROGRESS_PPBOOT_START = 100,
-	PROGRESS_PPBOOT_FINISH = 101,
-	PROGRESS_BOOT_ROM_IN_RECOVERY = 102,
-	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 103,
-	PROGRESS_BOOT_ROM_PCIE_LINK_START = 104,
-	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 105,
-	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 106,
-	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 107,
-	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 108,
-	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 109,
-	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 110,
-	PROGRESS_BOOT_ROM_D2D_LINK_START = 111,
-	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 112,
-	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 113,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 114,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 115,
-	PROGRESS_BOOT_ROM_RECOVERY_END = 116,
-	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 117,
-	PROGRESS_RECOVERY_IMG_IMG0 = 118,
-	PROGRESS_RECOVERY_IMG_IMG0_NO_IMG_DESC = 119,
-	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 120,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_STARTADDR_INVALID = 121,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_START = 122,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_CRC_FAILED = 123,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 124,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 125,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 126,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET = 127,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_ERROR = 128,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_FINISH = 129,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_VERIFICATION_FAIL = 130,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_START = 131,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_FINISH = 132,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START = 133,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH = 134,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START = 135,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH = 136,
-	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL = 137,
-	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL = 138,
-	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL = 139,
-	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL = 140,
-	PROGRESS_BOOT_INFO_LFS_FILE_NAME_FAIL = 141,
-	PROGRESS_BOOT_INFO_LFS_FINISHED = 142,
-	PROGRESS_BOOT_INFO_MANIFEST_CONTEXT_SET_FAIL = 143,
-	PROGRESS_BOOT_INFO_MANIFEST_HEADER_SET_FAIL = 144,
-	PROGRESS_BOOT_INFO_MANIFEST_CONTEXT_GET_FAIL = 145,
-	PROGRESS_BOOT_INFO_MANIFEST_HEADER_GET_FAIL = 146,
-	PROGRESS_BOOT_INFO_SVN_SET_FAIL = 147,
-	PROGRESS_BOOT_INFO_SVN_GET_FAIL = 148,
+	PROGRESS_INIT,
+	PROGRESS_INIT_ARC1_ICCM_INIT_FAILED,
+	PROGRESS_INIT_SPI_MANIFEST_0_EXCEEDS_MAX_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_START,
+	PROGRESS_INIT_SPI_MANIFEST_0_CRC_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_DEST,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_IMG_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_FLASH_OFFSET,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_ERROR,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_FINISH,
+	PROGRESS_INIT_SPI_MANIFEST_0_HDR_MAGIC_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_OFFSET,
+	PROGRESS_INIT_SPI_MANIFEST_0_VERIFICATION_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_INFO_SET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_DATA_SET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_INFO_GET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_DATA_GET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_0_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_EXCEEDS_MAX_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_START,
+	PROGRESS_INIT_SPI_MANIFEST_1_CRC_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_DEST,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_IMG_SIZE,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_FLASH_OFFSET,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_ERROR,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_FINISH,
+	PROGRESS_INIT_SPI_MANIFEST_1_HDR_MAGIC_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_OFFSET,
+	PROGRESS_INIT_SPI_MANIFEST_1_VERIFICATION_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_INFO_SET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_DATA_SET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_INFO_GET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_DATA_GET_FAIL,
+	PROGRESS_INIT_SPI_MANIFEST_1_FAIL,
+	PROGRESS_INIT_SPI_VALID_MANIFEST_INDEX_FAIL,
+	PROGRESS_INIT_EFUSE_INIT_FAIL,
+	PROGRESS_INIT_EFUSE_SPI,
+	PROGRESS_INIT_EFUSE_SPI_IMG_DESC_FAIL,
+	PROGRESS_INIT_EFUSE_SPI_EXCEEDS_MAX_SIZE,
+	PROGRESS_INIT_EFUSE_SPI_DMA_COPY_FAIL,
+	PROGRESS_INIT_EFUSE_SPI_MAGIC_FAIL,
+	PROGRESS_INIT_EFUSE_SPI_DESC_COUNT_FAIL,
+	PROGRESS_INIT_EFUSE_SPI_DECODE_FAIL,
+	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT,
+	PROGRESS_REVOKE_LOC_ASIC_EFUSE,
+	PROGRESS_REVOKE_LOC_FAKE_FUSE,
+	PROGRESS_KEY_LOC_ROM,
+	PROGRESS_VER_LOC_ASIC_EFUSE,
+	PROGRESS_VER_LOC_FAKE_FUSE,
+	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_START,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_VERSION_FAIL,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_REVOKED,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_HASH_FAIL,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_AUTH_FAIL,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_SUCCESS,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_START,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_VERSION_FAIL,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_REVOKED,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_HASH_FAIL,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_AUTH_FAIL,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_SUCCESS,
+	PROGRESS_BRINGUP_PCI_PRI,
+	PROGRESS_BRINGUP_PCI_PRI_NO_TMG_DESC,
+	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID,
+	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID,
+	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG,
+	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR,
+	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE,
+	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR,
+	PROGRESS_BRINGUP_PCI_PRI_DMA,
+	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL,
+	PROGRESS_BRINGUP_PCI_PRI_VERIFY_SUCCESS,
+	PROGRESS_BRINGUP_PCI_SEC,
+	PROGRESS_BRINGUP_PCI_SEC_NO_IMG_DESC,
+	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID,
+	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID,
+	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG,
+	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR,
+	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE,
+	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR,
+	PROGRESS_BRINGUP_PCI_SEC_DMA,
+	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL,
+	PROGRESS_BRINGUP_PCI_SEC_VERIFY_SUCCESS,
+	PROGRESS_BOOT_ROM_TPM,
+	PROGRESS_BOOT_ROM_TPM_INIT_FAIL,
+	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL,
+	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL,
+	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL,
+ 	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC,
+	PROGRESS_BOOT_ROM_TPM_FINISH,
+	PROGRESS_PPBOOT_START,
+	PROGRESS_PPBOOT_FINISH,
+	PROGRESS_BOOT_ROM_IN_RECOVERY,
+	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY,
+	PROGRESS_BOOT_ROM_PCIE_LINK_START,
+	PROGRESS_BOOT_ROM_PCIE_LINK_INIT,
+	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT,
+	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT,
+	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1,
+	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG,
+	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH,
+	PROGRESS_BOOT_ROM_D2D_LINK_START,
+	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY,
+	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS,
+	PROGRESS_BOOT_ROM_RECOVERY_END,
+	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG,
+	PROGRESS_RECOVERY_IMG_IMG0,
+	PROGRESS_RECOVERY_IMG_IMG0_NO_IMG_DESC,
+	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_STARTADDR_INVALID,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_START,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_CRC_FAILED,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_DEST,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_ERROR,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_FINISH,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_VERIFICATION_FAIL,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_START,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_FINISH,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH,
+	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL,
+	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL,
+	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL,
+	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL,
+	PROGRESS_BOOT_INFO_LFS_FILE_NAME_FAIL,
+	PROGRESS_BOOT_INFO_LFS_FINISHED,
+	PROGRESS_BOOT_INFO_MANIFEST_CONTEXT_SET_FAIL,
+	PROGRESS_BOOT_INFO_MANIFEST_HEADER_SET_FAIL,
+	PROGRESS_BOOT_INFO_MANIFEST_CONTEXT_GET_FAIL,
+	PROGRESS_BOOT_INFO_MANIFEST_HEADER_GET_FAIL,
+	PROGRESS_BOOT_INFO_SVN_SET_FAIL,
+	PROGRESS_BOOT_INFO_SVN_GET_FAIL,
 };
 
 #endif /* _PROGRESS_BITS_H_ */
